CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.7.0 (September 21, 2020)
+
+Since `1.7.0`, vscode-quarkus becomes an extension of [vscode-microprofile](https://github.com/redhat-developer/vscode-microprofile), see [CHANGELOG of vscode-microprofile](https://github.com/redhat-developer/vscode-microprofile/CHANGELOG.md#010-september-14-2020)
```

Comment:
vscode-microprofile will be automatically installed alongside vscode-quarkus.

Reply:
fixed